package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-webapp",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
Don't bump the version, I will do it, instead just let me know that you would like to see this feature published ASAP. 😉

Reply:
Oh! Great! I will update it now ^^